src/services/cacheConfigUtils.ts

```diff
@@ -146,4 +146,51 @@ export class CacheConfigUtils {
 
     return contentTypes;
   }
+
+  public static createCacheConfigFromData(data: CacheConfiguration): string {
+    let cache;
+
+    if (data.basic.topology === 'Distributed') {
+      cache = {
+        'distributed-cache': {
+          mode: 'SYNC',
+          owners: data.basic.numberOfOwners,
+          statistics: data.basic.statistics,
+          encoding: {
+            'media-type': data.basic.encoding,
+          },
+          locking: {
+            isolation: data.advanced.isolationLevel,
+          },
+          memory: {
+            'max-count': data.advanced.maxCount,
+            'when-full': data.advanced.evictionStrategy,
+          },
+        },
+      };
+    } else if (data.basic.topology === 'Replicated') {
+      cache = {
+        'replicated-cache': {
+          mode: 'SYNC',
+          statistics: data.basic.statistics,
+          encoding: {
+            'media-type': data.basic.encoding,
+          },
+          locking: {
+            isolation: data.advanced.isolationLevel,
+          },
+          memory: {
+            'max-count': data.advanced.maxCount,
+            'max-size': data.advanced.maxSize,
+            'when-full': data.advanced.evictionStrategy,
+          },
+          indexing: {
```

Comment:
@karesti Got an error when creating cache with Replicated Topology, regarding indexing storage 

![Screenshot from 2022-03-24 12-18-40](https://user-images.githubusercontent.com/97534722/159858919-d0a73b85-4503-4dac-b623-22fccbd075b3.png)

Reply:
the indexing part should not be present

Reply:
> Another thing I think we must change is the expiration part. We put that in the advanced options but it's a cache feature : "bounded". The expiration option should not be there. When we will add the features we will put that back. Sorry about that

Careful: Bounded is Eviction, not Expiration.